Autopilot/Inc/SensorFusion.hpp

```diff
@@ -30,6 +28,31 @@ struct SFOutput_t {
     float longitudeSpeed; //m/s
 };
 
+
+#ifndef ATTITUDE_DATATYPES_HPP //Temporary protection until redefinitions are deleted
```

Comment:
Chief's PR that we will try to merge soon will remove these structs from the AttitudeDatatypes.hpp. As a result, I would remove this compile guard, and you can probably rename the structs if you'd like to prevent clashes in the meantime:

`IMU_Data_t` --> `IMU_Raw_Data_t`
`Airspeed_Data_t` --> `Airspeed_Raw_Data_t`

Also, you could make structs and functions for the GPS and altimeter too. That way you don't have to make another PR for those sensors in the future lol.